cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -412,13 +390,13 @@ func metricsListGrabAggregateConvert(t metricsTarget) (map[string]*dto.MetricFam
 		origFamilies, err := parser.TextToMetricFamilies(bytes.NewReader(rawOutput))
 
 		if err != nil {
-			glog.Warning("Failed to parse metrics:", err)
+			log.Error("Failed to parse metrics:")
 			return nil, nil, true, err
 		}
 
 		families, err = filterMetrics(origFamilies, interestingMetrics)
 		if err != nil {
-			glog.Warning("Failed to filter metrics:", err)
+			log.Error("Failed to filter metrics:")
```

Comment:
remove this. Just let the error bubble up to the caller and the caller can log a message if it wants.

Reply:
Ack.